io.github.chilledheart.yass.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.chilledheart.yass</id>
+  <name>YASS</name>
+  <developer id="letshack.info">
```

Comment:
this should be reverse DNS too. Is this your website?

Reply:
Done.

Reply:
1157509fa513f001c0e627d9ea1795ec76484527